### PR TITLE
Clarify documentation of the age parameter

### PR DESCRIPTION
**Pull request by techcobweb**
Signed-off-by: Mike Cobbett <77053+techcobweb@users.noreply.github.com>

### CODE REVIEW

File: docs/generated/galasactl_runs_get.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ galasactl runs get [flags]
 ### Options
 
 ```
-      --age string         the age of the test run(s) we want information about. Supported formats are: 'FROM' or 'FROM:TO', where FROM and TO are each ages, made up of an integer and a time-unit qualifier. Supported time-units are 'w' (weeks), 'd' (days), 'h' (hours), 'm' (minutes). If missing, the TO part is defaulted to '0h'. Examples: '--age 1d' , '--age 6h:1h' 
+      --age string         the age of the test run(s) we want information about. Supported formats are: 'FROM' or 'FROM:TO', where FROM and TO are each ages, made up of an integer and a time-unit qualifier. Supported time-units are 'w' (weeks), 'd' (days), 'h' (hours), 'm' (minutes). If missing, the TO part is defaulted to '0h'. Examples: '--age 1d' , '--age 6h:1h'.The TO part must be a smaller time-span than the FROM part.
```
Comment (eamansour) [inline docs/generated/galasactl_runs_get.md:16]: Looks like the last sentence could do with a space after the "." at the start?

**Review by eamansour — APPROVED** [review]
Looks good - just have a minor formatting suggestion :)

**Review by eamansour — APPROVED** [review]

**Comment by galasa-team** [comment]
Build successful

**Comment by galasa-team** [comment]
Build successful

**Comment by galasa-team** [comment]
Build successful

**Pull request merged**